pkg/policy/chain_test.go

```diff
@@ -111,113 +107,20 @@ func TestEvaluateCertificateRequest(t *testing.T) {
 		},
 	}
 
-	for name, test := range tests {
+	for name, tc := range tests {
```

Comment:
```suggestion
	for name, test := range tests {
```

Reply:
Fixed in c1b96d5

Reply:
This is the name of a package. Any other suggestions?

Reply:
By removing the renaming of the gen package we can use test here.